content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -784,15 +784,7 @@ registries:
 
 ### `docker-registry`
 
-{% data variables.product.prodname_dependabot %} works with container registries that implement the OCI container registry. For more information, see [https://github.com/opencontainers/distribution-spec/blob/main/spec.md](https://github.com/opencontainers/distribution-spec/blob/main/spec.md).  {% data variables.product.prodname_dependabot %} supports authentication to private registries via a central service. For further details, see [Token Authentication Specification](https://docs.docker.com/registry/spec/auth/token/) in the Docker documentation.
-
-We currently support the container registries listed here:
-
-* Docker Hub
-* {% data variables.product.company_short %} {% data variables.product.prodname_container_registry %}
-* GCR (Google Cloud)
-* Private ECR (AWS) - public ECR support is tracked in [https://github.com/dependabot/dependabot-core/issues/4212](https://github.com/dependabot/dependabot-core/issues/4212).
-
+{% data variables.product.prodname_dependabot %}  works with any container registries that implement the OCI container registry spec. For more information, see [https://github.com/opencontainers/distribution-spec/blob/main/spec.md](https://github.com/opencontainers/distribution-spec/blob/main/spec.md).  {% data variables.product.prodname_dependabot %} supports authentication to private registries via a central token service or via HTTP Basic Auth. For further details, see [https://docs.docker.com/registry/spec/auth/token/](https://docs.docker.com/registry/spec/auth/token/) and [https://en.wikipedia.org/wiki/Basic_access_authentication](https://en.wikipedia.org/wiki/Basic_access_authentication).
```

Comment:
@mchammer01 I was unclear when I should be using an inline link vs not... since over in https://github.com/github/docs/pull/20651/files#diff-cbce3eccf5f535f4502688e4253bd81c56841099d6feb271228f9ed403b25efcR779 you said not to use inline links, but then hyperlinked `Token Authentication Specification`... so just left me confused 😄 

Anyway, feel free to tweak the link structure/wording on this PR as needed... we just somehow need to communicate that we support/require OCI Auth and Basic Auth for private container registries.

Reply:
@jeffwidman - will review. Yes, sorry, no inline links in general, especially when linking internally to our own docs.